README.md

```diff
@@ -1,8 +1,162 @@
 # trading_calendars
 
 ![CI](https://github.com/quantopian/trading_calendars/workflows/CI/badge.svg)
+[![PyPI version](https://badge.fury.io/py/trading-calendars.svg)](https://badge.fury.io/py/trading-calendars)
 
-A Python library of exchange calendars meant to be used with [Zipline](https://github.com/quantopian/zipline).
+A Python library of exchange calendars, frequently used with [Zipline](https://github.com/quantopian/zipline).
+
+
+## Installation
+
+```bash
+$ pip install trading-calendars
+```
+
+## Quick Start
+
+```python
+import trading_calendars as tc
+import pandas as pd
+import pytz
+```
+
+Get all registered calendars with `get_calendar_names`:
+
+```python
+tc.get_calendar_names()[:5]
+```
+    ['XPHS', 'FWB', 'CFE', 'CMES', 'XSGO']
+
+Instantiate a calendar with `get_calendar`:
+
+```python
+xnys = tc.get_calendar("XNYS")
+```
+
+Working with sessions:
+
+```python
+xnys.is_session(pd.Timestamp("2020-01-01"))
+```
+
+    False
```

Comment:
How about we format the results as python too?

Reply:
The results aren't python?

But I can put it in one big python blob with `>>>` prefixing outputs..

Reply:
Oh, I would expect them to be python, since it's the repr of each output, which is meant to be valid python to reconstruct that object. Aren't `False` and `Timestamp('2020-01-02 14:31:00+0000', tz='UTC')` python?